### PR TITLE
ejs changes and Refactoring  api's to server

### PR DESCRIPTION
**Pull request by cnallam**
…ogic to serverside, added issue templates

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -16,20 +16,25 @@
       "onAutoForward": "openBrowser"
     }
   },
+
   "secrets": {
     "CLIENT_ID": {
-      "description": "Sandbox client ID of the application.",
+      "description": "Sandbox client ID of the application. Leave blank to use default credentials",
```
Comment (rekhaabaidipati) [inline .devcontainer/devcontainer.json:22]: The new text suggested by content team is this 
`Sandbox client ID of the application.Uses default credential if none provided.`
`Sandbox secret of the application.Uses default credential if none provided.`

**Review by rekhaabaidipati — COMMENTED** [review]
Hint text suggestion from content team

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (cnallam) [reply .devcontainer/devcontainer.json]: Addressed

**Review by sdarshale — APPROVED** [review]
LGTM

**Pull request merged**